docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/LogisticRegressionWithOptions.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Trainers;
+
+namespace Microsoft.ML.Samples.Dynamic.Trainers.BinaryClassification
+{
+    public static class LogisticRegressionWithOptions
```

Comment:
>LogisticRegressionWithOptions [](start = 24, length = 29)

please make sure the sample is added to the xml documentation for BinaryClassification.Trainers.LbfgsLogisticRegression.